config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-4.yaml

```diff
@@ -92,10 +92,10 @@ presubmits:
         resources:
           requests:
             cpu: "10"
-            memory: "40Gi"
+            memory: "24Gi"
```

Comment:
cc @tenzen-y @alculquicondor for kueue.

Reply:
Thanks for the heads up. We reduced the limits in future versions. Changing it for the older version sgtm.

Reply:
LGTM
Thanks!